Resources/Prototypes/Entities/Objects/Fun/error.yml

```diff
@@ -4,6 +4,12 @@
   name: error
   description: Hmmmm. Something went wrong.
   components:
+  - type: Item
+    shape: # funny shape
+    - 0,0,6,0
+    - 0,1,0,2
+    - 0,3,6,3
+    - 6,1,6,2
```

Comment:
due to the limits of the grid piece rendering, i would prefer if there wasn't  a gap in the center of the shape. It looks extremely buggy and will be covered up, functionally obscuring it. Maybe just do it like so:
```
XXXXXXX
X.XXX.X
X.XXX.X
XXXXXXX
```

again, this is another one where i'm not really sure what the purpose is. it's fine if it's just for gaffs but it has to look good. Then again, the shape is also really big so i don't even know if this is something we want.

Reply:
It is obscured if other items are added to the bag, but it doesn't prevent function.
The error can be moved by clicking on the grid, and items placed in the middle can be interacted with as usual.

There's no reason to it, its just a funny error item.